src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/LocationBasedServicesAccount/GetAzureLocationBasedServicesAccountKey.cs

```diff
@@ -0,0 +1,84 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.LocationBasedServices.Models;
+using Microsoft.Azure.Management.LocationBasedServices;
+using Microsoft.Azure.Management.LocationBasedServices.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.LocationBasedServices
+{
+    /// <summary>
+    /// Get Account Keys for Location Based Services Account
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, LocationBasedServicesAccountKeyNounStr), OutputType(typeof(PSLocationBasedServicesAccountKeys))]
+    public class GetAzureLocationBasedServicesAccountKeyCommand : LocationBasedServicesAccountBaseCmdlet
+    {
+        protected const string NameParameterSet = "NameParameterSet";
+        protected const string InputObjectParameterSet = "InputObjectParameterSet";
+
```

Comment:
Needs the resource id parameter set as well

Reply:
ok.  refactored this a bit